tests/pxScene2d/test_imagecache.cpp

```diff
@@ -518,7 +518,7 @@ class rtHttpCacheTest : public testing::Test, public commonTestFns
       const char* cacheData = "This is image data";
```

Comment:
Please create new unit tests in test_imagecache.cpp to test the new APIs.  Adding code coverage for the new APIs will ensure our code coverage does not drop.